#Request
float64 effort
---
#Response
bool success